cmd/agent/agent.go

```diff
@@ -460,6 +463,95 @@ func serverWSCommunication(config config.Config, messages chan *streaming.WSMess
 	}
 }
 
+func kubeEvents(kubeEnv *agent.KubeEnv, gimletHost string, agentKey string) {
+	svc, err := kubeEnv.Client.CoreV1().Services("").List(context.TODO(), meta_v1.ListOptions{})
+	if err != nil {
+		log.Errorf("could not get services: %v", err)
+		return
+	}
+
+	var integratedServices []v1.Service
+	for _, s := range svc.Items {
```

Comment:
Why don't we use https://github.com/gimlet-io/gimlet/blob/99020ee94bc4da5274d2f39501042ca7e6fe992b/pkg/agent/kube.go#L86? I think this is the canonical way to find services that are integrated with Gimlet.